libraries/cyclestreets-view/src/main/res/values/strings.xml

```diff
@@ -183,11 +183,13 @@
   <string name="remaining">Remaining</string>
   <string name="slash_ETA">/ ETA</string>
 
-  <string name="duration" translatable="false">Duration</string>
-  <string name="distance" translatable="false">Distance</string>
-  <string name="circular_route" translatable="false">Circular Route</string>
-  <string name="create_circular_route_desc" translatable="false">Create a circular route based on a duration or distance</string>
-  <string name="create_circular_route_button" translatable="false">Create circular route</string>
-  <string name="mins" translatable="false">mins</string>
-
+  <string name="duration">Duration</string>
+  <string name="distance">Distance</string>
+  <string name="circular_route">Circular Route</string>
+  <string name="create_circular_route_desc">Create a circular route based on a duration or distance</string>
+  <string name="create_circular_route_button">Create circular route</string>
+  <string name="mins">mins</string>
+  <string name="choose_pois_to_route_via">Optionally, choose Points of Interest to route via:</string>
```

Comment:
Would this be better as `Optionally, choose Point of Interest types via which to route:`?

Reply:
(same applies to all untranslated translations, obviously)

Reply:
While the text in the code isn't perfect queen's english grammar, I think in practice it is clearer.

Reply:
Sacrilege! 😂 Fair enough, I'm not hung up on it.

Reply:
Phew! ;-)  I haven't yet found an easy way to amend all the translations, esp without spaces getting added / removed in the rest of the file and making it look like loads of lines have been amended - I'm sure there must be a way.